src/gui/search/searchjobwidget.cpp

```diff
@@ -513,13 +509,21 @@ void SearchJobWidget::appendSearchResults(const QVector<SearchResult> &results)
         int row = m_searchListModel->rowCount();
         m_searchListModel->insertRow(row);
 
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::NAME), result.fileName); // Name
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::DL_LINK), result.fileUrl); // download URL
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::SIZE), result.fileSize); // Size
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::SEEDS), result.nbSeeders); // Seeders
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::LEECHES), result.nbLeechers); // Leechers
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::ENGINE_URL), result.siteUrl); // Search site URL
-        m_searchListModel->setData(m_searchListModel->index(row, SearchSortModel::DESC_LINK), result.descrLink); // Description Link
+        const auto setModelData = [this, row] (const int column, const QString &displayData
+                                         , const QVariant &underlyingData, const QVariant &textAlignmentData = {})
+        {
+            m_searchListModel->setData(m_searchListModel->index(row, column), displayData, Qt::DisplayRole);
+            m_searchListModel->setData(m_searchListModel->index(row, column), underlyingData, SearchSortModel::UnderlyingDataRole);
+            m_searchListModel->setData(m_searchListModel->index(row, column), textAlignmentData, Qt::TextAlignmentRole);
+        };
+
+        setModelData(SearchSortModel::NAME, result.fileName, result.fileName);
```

Comment:
Dunno why data layout is defined in sort model. It should be defined in underlying model.